src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -202,11 +201,19 @@ object LoadVCF {
     dropSamples: Boolean = false,
     gr: GenomeReference = GenomeReference.GRCh37): VariantSampleMatrix[Locus, Variant, Annotation] = {
     val sc = hc.sc
+    val hConf = hc.hadoopConf
+    val headerLines1 = getHeaderLines(hConf, file1)
+    val header1 = parseHeader(reader, headerLines1)
+    val header1Bc = sc.broadcast(header1)
+
+    val conf = new SerializableHadoopConfiguration(hConf)
```

Comment:
broadcast this - the conf parameters are super slow to deserialize, so we want to do that once per JVM instead of once per partition.